android/AndroidSportsPlatform/app/src/main/java/com/example/sportsplatform/data/UserRepository.kt

```diff
@@ -7,7 +7,7 @@ class UserRepository(private val api: UserApi) {
     suspend fun findUser(userRequest: UserRequest) : Response<TokenResponse> {
         return api.searchUser(userRequest)
     }
-    suspend fun signUser(userRegisterRequest: UserRegisterRequest) : Response<UserRegisterResponse> {
+    suspend fun signUser(userRegisterRequest: UserRegisterRequest) : Response<String> {
         return api.registerUser(userRegisterRequest)
```

Comment:
Since the user register responso is an empty string, we should delete "UserRegisterResponse" model and create another model named "User".
"User" model should include the necessary fields for a User which could be discussed later.